[2024-04-16T07:04:54.097+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T07:04:54.142+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_trigger_sync2 scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-16T07:04:54.184+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_trigger_sync2 scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-16T07:04:54.185+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-16T07:04:54.268+0000] {taskinstance.py:2327} INFO - Executing <Task(AirbyteTriggerSyncOperator): airbyte_trigger_sync2> on 2024-04-15 00:00:00+00:00
[2024-04-16T07:04:54.300+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=55) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T07:04:54.302+0000] {standard_task_runner.py:63} INFO - Started process 60 to run task
[2024-04-16T07:04:54.315+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'trigger_airbyte_job_example', 'airbyte_trigger_sync2', 'scheduled__2024-04-15T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/dag_aribyte_ex.py', '--cfg-path', '/tmp/tmpvnuuumr3']
[2024-04-16T07:04:54.321+0000] {standard_task_runner.py:91} INFO - Job 36: Subtask airbyte_trigger_sync2
[2024-04-16T07:04:54.566+0000] {task_command.py:426} INFO - Running <TaskInstance: trigger_airbyte_job_example.airbyte_trigger_sync2 scheduled__2024-04-15T00:00:00+00:00 [running]> on host ff62e46d2b31
[2024-04-16T07:04:54.952+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='trigger_airbyte_job_example' AIRFLOW_CTX_TASK_ID='airbyte_trigger_sync2' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-15T00:00:00+00:00'
[2024-04-16T07:04:54.959+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T07:04:54.991+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-16T07:04:57.172+0000] {airbyte.py:86} INFO - Job 160 was submitted to Airbyte Server
[2024-04-16T07:04:57.175+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T07:04:57.274+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=trigger_airbyte_job_example, task_id=airbyte_trigger_sync2, execution_date=20240415T000000, start_date=20240416T070454, end_date=20240416T070457
[2024-04-16T07:04:57.371+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T07:04:57.434+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-16T07:04:57.448+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
